### PR TITLE
Add UserGroup resource

### PR DESCRIPTION
**Pull request by kgoodwin-mdl**

### CODE REVIEW

File: lib/dock_health_api/resources/usergroup.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      extend DockHealthApi::Crud::Get
+      extend DockHealthApi::Crud::Update
+      extend DockHealthApi::Crud::List
+  
+      def self.resource_url
+        "#{client.config.resource_url}/api/#{url_version}/user/group"
+      end
+
+    end
+  end
```
Comment (lewismjones) [inline lib/dock_health_api/resources/usergroup.rb:14]: what is this indentation 😂

Comment (lewismjones) [reply lib/dock_health_api/resources/usergroup.rb:14]: @kgoodwin-mdl , do we need all of these endpoints for usergroups?

**Review by lewismjones — CHANGES_REQUESTED** [review]

File: lib/dock_health_api/resources/usergroup.rb (hunk 1 above)
Comment (lewismjones) [reply lib/dock_health_api/resources/usergroup.rb:14]: less endpoints means less specs 😎

**Comment by lewismjones** [comment]
@kgoodwin-mdl don't forget to bump the version #

File: lib/dock_health_api/resources/usergroup.rb (hunk 1 above)
Comment (kgoodwin-mdl) [reply lib/dock_health_api/resources/usergroup.rb:14]: Ill fix that, it was copy paste error from the other file.

Not sure if we need all of them but they are implemented by DH

**Review by Evilfishbone — APPROVED** [review]

**Review by lewismjones — APPROVED** [review]

**Pull request merged**